project_euler/problem_42/sol1.py

```diff
@@ -0,0 +1,44 @@
+"""
+The nth term of the sequence of triangle numbers is given by, tn = ½n(n+1); so the first ten triangle numbers are:
+
+1, 3, 6, 10, 15, 21, 28, 36, 45, 55, ...
+
+By converting each letter in a word to a number corresponding to its alphabetical position and adding these values we form a word value. For example, the word value for SKY is 19 + 11 + 25 = 55 = t10. If the word value is a triangle number then we shall call the word a triangle word.
+
+Using words.txt (right click and 'Save Link/Target As...'), a 16K text file containing nearly two-thousand common English words, how many are triangle words?
+"""
+import os
+
+
+# Precomputes a list of the 100 first triangular numbers
+TRIANGULAR_NUMBERS = set(map(lambda n: int(0.5 * n * (n + 1)), range(1, 101)))
```

Comment:
TRIANGULAR_NUMBERS = set(int(0.5 * n * (n + 1)) for n in range(1, 101)) as discussed in CONTRIBUTING.md